src/source/Signaling/Signaling.c

```diff
@@ -15,11 +15,12 @@ STATUS createSignalingSync(PSignalingClientInfoInternal pClientInfo, PChannelInf
     struct lws_context_creation_info creationInfo;
     PStateMachineState pStateMachineState;
     BOOL cacheFound = FALSE;
-    SignalingFileCacheEntry fileCacheEntry;
+    PSignalingFileCacheEntry pFileCacheEntry = NULL;
 
     CHK(pClientInfo != NULL && pChannelInfo != NULL && pCallbacks != NULL && pCredentialProvider != NULL && ppSignalingClient != NULL,
         STATUS_NULL_ARG);
     CHK(pChannelInfo->version <= CHANNEL_INFO_CURRENT_VERSION, STATUS_SIGNALING_INVALID_CHANNEL_INFO_VERSION);
+    CHK(NULL != (pFileCacheEntry = (PSignalingFileCacheEntry) MEMALLOC(sizeof(SignalingFileCacheEntry))), STATUS_NOT_ENOUGH_MEMORY);
```

Comment:
Could you change sizeof -> SIZEOF

Reply:
Sorry for the mistake. pushed it just now.